src/test/java/com/amadeus/NamespaceTest.java

```diff
@@ -90,12 +87,8 @@ public class NamespaceTest {
     assertNotNull(client.shopping.flightOffersSearch.pricing);
     assertNotNull(client.shopping.flightOffers.prediction);
     assertNotNull(client.shopping.flightOffers.upselling);
-    assertNotNull(client.shopping.hotelOffers);
-    assertNotNull(client.shopping.hotelOffersSearch);
```

Comment:
HotelOffersSearch is for Hotel Search V3, no? :astonished:

Reply:
good catch :O